x-pack/test_serverless/functional/test_suites/common/management/advanced_settings.ts

```diff
@@ -22,6 +22,7 @@ const editorSettings = new Set<string>([
   settings.SECURITY_SOLUTION_RULES_TABLE_REFRESH_ID,
   settings.SECURITY_SOLUTION_IP_REPUTATION_LINKS_ID,
   settings.OBSERVABILITY_APM_AWS_LAMBDA_PRICE_FACTOR_ID,
+  settings.AI_ANONYMIZATION_SETTINGS,
```

Comment:
This settings along with other AI Assistant ones don't show in the Search Advanced Settings only in the AI Assistant settings. So I added this here so it skips the check.  I'm not sure why it fails on this particular setting and not the others.